src/components/textRe.re

```diff
@@ -43,59 +40,60 @@ module CreateComponent (Impl: ViewRe.Impl) :TextComponent => {
-          "onLongPress": from_opt onLongPress,
-          "onPress": from_opt onPress,
-          "pressRetentionOffset": from_opt pressRetentionOffset,
-          "selectable": from_opt (UtilsRN.optBoolToOptJsBoolean selectable),
-          "style": from_opt style,
-          "testID": from_opt testID,
-          "selectionColor": from_opt selectionColor,
-          "textBreakStrategy":
-            from_opt (
-              UtilsRN.option_map
-                (
-                  fun
-                  | `simple => "simple"
-                  | `highQuality => "highQuality"
-                  | `balanced => "balanced"
-                )
-                textBreakStrategy
-            ),
-          "adjustsFontSizeToFit": from_opt (UtilsRN.optBoolToOptJsBoolean adjustsFontSizeToFit),
-          "minimumFontScale": from_opt minimumFontScale,
-          "suppressHighlighting": from_opt (UtilsRN.optBoolToOptJsBoolean suppressHighlighting)
-        }
-      )
-      children::(
+      children =>
+    ReasonReact.wrapJsForReason
+      reactClass::Impl.view
+      props::
+        Js.Undefined.(
+          {
+            "accessible": from_opt (UtilsRN.optBoolToOptJsBoolean accessible),
+            "allowFontScaling": from_opt (UtilsRN.optBoolToOptJsBoolean allowFontScaling),
+            "ellipsizeMode":
+              from_opt (
+                UtilsRN.option_map
+                  (
+                    fun
+                    | `head => "head"
+                    | `middle => "middle"
+                    | `tail => "tail"
+                    | `clip => "clip"
+                  )
+                  ellipsizeMode
+              ),
+            "numberOfLines": from_opt numberOfLines,
+            "onLayout": from_opt onLayout,
+            "onLongPress": from_opt onLongPress,
+            "onPress": from_opt onPress,
+            "pressRetentionOffset": from_opt pressRetentionOffset,
+            "selectable": from_opt (UtilsRN.optBoolToOptJsBoolean selectable),
+            "style": from_opt style,
+            "testID": from_opt testID,
+            "selectionColor": from_opt selectionColor,
+            "textBreakStrategy":
+              from_opt (
+                UtilsRN.option_map
+                  (
+                    fun
+                    | `simple => "simple"
+                    | `highQuality => "highQuality"
+                    | `balanced => "balanced"
+                  )
+                  textBreakStrategy
+              ),
+            "adjustsFontSizeToFit": from_opt (UtilsRN.optBoolToOptJsBoolean adjustsFontSizeToFit),
+            "minimumFontScale": from_opt minimumFontScale,
+            "suppressHighlighting": from_opt (UtilsRN.optBoolToOptJsBoolean suppressHighlighting)
+          }
+        )
+      (
         switch value {
-        | Some string => [ReactRe.stringToElement string, ...children]
+        | Some string => Array.append [|ReasonReact.stringToElement string|] children
```

Comment:
The `children` which are passed to `make` are now `array reactElement` and no more `list reactElement`